Dockerfile

```
@@ -0,0 +1,13 @@
+FROM golang:1.22.5
+
+WORKDIR /app
+
+COPY go.mod go.sum tracker.db ./
+
+RUN go mod tidy && go mod download
```

`tidy` проходиться по коду и ищет зависимости, а тут кода нет
```suggestion
RUN go mod download
```